test/new_tests/test_exists_many.py

```diff
@@ -25,6 +27,20 @@ def test_pos_exists_many_without_policy(self, put_data):
         assert isinstance(records, list)
         assert len(records) == rec_length
 
+    def test_pos_exists_many_with_proper_parameters_without_connection(self, put_data):
+        self.keys = []
+        rec_length = 5
+        for i in range(rec_length):
+            key = ("test", "demo", i)
+            record = {"name": "name%s" % (str(i)), "age": i}
+            put_data(self.as_connection, key, record)
+            self.keys.append(key)
+        records = self.as_connection.exists_many(self.keys, {"total_timeout": 1200})
+
+        assert isinstance(records, list)
+        assert len(records) == rec_length
+        assert Counter([x[0][2] for x in records]) == Counter([0, 1, 2, 3, 4])
+
```

Comment:
Shouldn't this test check if `exists_many()` fails after calling `close()`?

Reply:
its existing test case, actual neg test case is below